.circleci/config.yml

```diff
@@ -76,15 +72,6 @@ jobs:
           name: Upload coverage
           command: bash <(curl -s https://codecov.io/bash)
 
-  test-build:
```

Comment:
Don't move this to github actions please, I don't see GH actions replacing this yet